File: e2e/selenide-junit5-selenium-grid/src/test/java/test/mobile/createTask/CreateTaskTest.java
package test.mobile.createTask;

import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.TmsLink;
import test.base.MobileTest;
import test.spec.createTask.ICreateTaskTest;

import static data.AllureEpic.TASK_MANAGEMENT;

@Epic(TASK_MANAGEMENT)
@Feature("Create task")
@TmsLink("100")
class CreateTaskTest extends MobileTest implements ICreateTaskTest {}

File: e2e/maestro/tests/create-task/create-task.test.ts
import { AllureEpic } from '@/data/AllureEpic';
import { maestroTestCase } from '@/runner/maestroTestCase';

export default maestroTestCase(import.meta.url, {
  name: 'should create task when form submitted with valid data',
  epic: AllureEpic.TASK_MANAGEMENT,
  feature: 'Create task',
  tms: '100',
  suite: 'mocked',
  setupMocks: async (support, context) => {
    const response = support.mock.api.fromResponse(context.createTaskResponse());

    await support.reset();
    await support.mock.api.getTasks(response);
    await support.mock.api.createTask(response);
    await support.mock.api.getTasks(response);
    await support.mock.api.getTasks(response);
    await support.mock.api.getTasks(response);
    await support.mock.api.getTask(response);
    await support.mock.api.getIsValid(true);

    return context.toMaestroEnv();
  },
});
